keps/sig-api-machinery/2896-openapi-v3/README.md

```diff
@@ -206,8 +208,19 @@ endpoints.
 
 #### Future Work
 
-After OpenAPI v3 is implemented, we can start working on updating clients to
-consume OpenAPI v3. A separate KEP will be published for that.
+After OpenAPI v3 is implemented, we can start working on updating
+clients to consume OpenAPI v3. A separate KEP will be published for
+them. One example is [OpenAPI V3 for kubectl
+explain](https://github.com/kubernetes/enhancements/blob/master/keps/sig-cli/3515-kubectl-explain-openapiv3/README.md).
+
+#### OpenAPI V2 plan
+
+As OpenAPI V3 becomes more and more available and easily accessible,
+clients should move to OpenAPI V3 in favor of OpenAPI V2. This should
+result in a steady decline in the number of requests sent to the
+OpenAPI V2 endpoint. In the short term, we will make OpenAPI V2 make
+computations more lazily, effectively using almost no compute until a
+request to the `/openapi/v2` endpoint is sent.
```

Comment:
That's a little hand-wavy, can we clarify what this mean to make it more lazy? is this part of GA or is this part of future work?